config/config_mgmt.py

```diff
@@ -2,8 +2,11 @@
 config_mgmt.py provides classes for configuration validation and for Dynamic
 Port Breakout.
 '''
+
+import os
 import re
 import syslog
+import yang as ly
```

Comment:
Why abbreviate to `ly`? Only saves two chars and makes it less readable.

Reply:
I agree but I stick to existing code style/naming, I took this import basically from - https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-mgmt/sonic_yang.py#L1.